tests/MakeCommandTest.php

```diff
@@ -0,0 +1,41 @@
+<?php
+namespace Tests;
+
+use PHPUnit_Framework_TestCase as TestCase;
```

Comment:
Use `PHPUnit\Framework\TestCase` here; 5.7 adds forwards-compat shims to make that work, and 6.0 only has namespaced classes.